bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/GmosSDtaXRuleTest.scala

```diff
@@ -71,6 +78,11 @@ class GmosSDtaXRuleTest extends AbstractRuleTest {
 
   private def expectFailure2(): Unit = expectFailure("Ybin=2")
 
+  private def expectFailure3(): Unit =
+    assertTrue (runCheck.exists { p =>
+      p.getId() == "GmosRule_DTA_X_Y_MULTIPLE_BINNING_RULE"
```

Comment:
We could expose the problem id in the corresponding GMOS rule so we can just reference it from the test.